README.md

```diff
@@ -1,6 +1,6 @@
 # partial_sort
 
-[![](https://img.shields.io/crates/v/logforth.svg)](https://crates.io/crates/logforth)
```

Comment:
seems you are using logforth..

Reply:
So we correct in this PR. This is why the title include "fix README".